test/test_hparams.jl

```diff
@@ -88,4 +88,19 @@ end
         decoded_v = hv.kind.value
         @test expected_hparams_config[k] == decoded_v
     end
+end
+
+@testset "Reading with hparams present (#137)" begin
+    # https://github.com/JuliaLogging/TensorBoardLogger.jl/issues/137
+    lg = TBLogger(joinpath(mktempdir(), "logs"))
+    TensorBoardLogger.write_hparams!(
+        lg,
+        Dict("hi" => 1.0),
+        ["x/val"]
+    )
+    with_logger(lg) do
+        @info "x" val=3.0
+    end
+    hist = convert(MVHistory, lg)
```

Comment:
this is the line that is failing. Is what we need:
```suggestion
    hist = convert(ValueHistories.MVHistory, lg)
```
Or do we need to make sure ValueHistories.jl is loaded?

Reply:
Ah I think I've gotten confused by the fact that ValueHistories.jl is listed as a dep for the `test` target in the Project.toml, and thus assumed it was already loaded. I loaded it in `runtests.jl` now:) Should hopefully do the trick.